sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/CHANGELOG.md

```diff
@@ -10,6 +10,17 @@
 
 ### Other Changes
 
+## 1.1.0 (2023-01-30)
```

Comment:
Please give two weeks buffer when trying to specify the release date.
Please remove the content of 1.1.0-beta.1 above.

Reply:
removed and updated release date as discussed